pkgs/by-name/cl/cloudflare-ddns/package.nix

```
@@ -0,0 +1,74 @@
+{
+  lib,
+  stdenv,
```

```suggestion
  stdenvNoCC,
```

C compiler not needed